sections/footer.liquid

```diff
@@ -47,7 +47,7 @@
   -%}
   {%- if section.blocks.size > 0
     or section.settings.newsletter_enable
-    or section.settings.show_social
+    or section.settings.show_social and has_social_icons == true
     or section.settings.enable_follow_on_shop
```

Comment:
Looking good 👍 

I think there is still a scenario in which it happens. When `enable_follow_on_shop` is enabled it doesn't always mean it's going to show the icon. There are some eligibility requirements: 
![](https://screenshot.click/28-45-qgas1-1cgoz.png)

But we can't really do anything ourselves to check on that. 
The follow on shop setting does mention having the Shop app enabled and links to the docs so I think that should be enough.